test/integration/spec/connect.js

```diff
@@ -203,6 +204,119 @@ describe('connect', function() {
     });
   });
 
+  describe.only('media region', () => {
+    let sid = null;
+    let token = null;
+    let roomName = null;
+
+    beforeEach(async () => {
+      const identity = randomName();
+      token = getToken(identity);
+      sid = await createRoom(randomName(), defaults.topology, { MediaRegion: 'gll' });
+    });
```

Comment:
I guess it will not be possible to test Ad-hoc Room creation unless the account under test has "gll" selected by default. Should we be adding tests for it? I did not add coverage of Ad-Hoc in C++ because of the complexity of having tests require credentials for more than 1 account SID.

Reply:
@ceaglest Yes. We would have to manually change the default media region in the console for the Account SID.